getting-started/c++/obstacle-avoiding-bot-c++/obstacle-avoiding-bot-c++.md

```diff
@@ -33,16 +33,16 @@ To program the component
 	cd src
 	gedit specificworker.cpp
 
-Here we will be using both the interfaces we have learnt so far. The entire algorithm is as follows
+Here we will be using both the interfaces we have learned so far. The entire algorithm is as follows
```

Comment:
https://www.grammar.com/learned_vs._learnt

Reply:
@orensbruli Sorry for the inconvenience. I am doing the changes.